sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormSubModel.cs

```diff
@@ -18,17 +18,21 @@ internal CustomFormSubModel(string formType, float? accuracy, IReadOnlyDictionar
         }
 
         /// <summary>
-        ///  Type of form this submodel recognizes.
+        /// The type of form this submodel recognizes.
         /// </summary>
         public string FormType { get; internal set; }
 
         /// <summary>
-        /// The mean of the model's field accuracies.
+        /// The mean of the accuracies of this model's <see cref="CustomFormModelField"/>
+        /// instances.
         /// </summary>
         public float? Accuracy { get; internal set; }
 
         /// <summary>
-        /// Form fields that this submodel will extract when analyzing this form type.
+        /// A dictionary of the key-value pairs that this submodel will recognize from the
+        /// input document. Labeled models will use pre-defined labels as keys, and a unique
+        /// key will be generated for <see cref="CustomFormModelField"/> instances with no
+        /// pre-defined label available.
```

Comment:
Let's say something like: the Fields dictionary keys are the Name of the field.  For models trained with labels, this is the training-time label of the field.  For models trained with forms only, a unique name is generated for each field.